### PR TITLE
Add clarification about 7-bit addressing in I2C.

### PR DESCRIPTION
**Pull request by ntoll**

### CODE REVIEW

File: docs/i2c.rst [hunk 1]
```diff
@@ -8,6 +8,10 @@ using the I²C bus protocol. There can be multiple slave devices connected at
 the same time, and each one has its own unique address, that is either fixed
 for the device or configured on it. Your board acts as the I²C master.
 
+We use 7-bit addressing for devices because of the reasons stated
+`here <http://www.totalphase.com/support/articles/200349176-7-bit-8-bit-and-10-bit-I2C-Slave-Addressing>`_. This maybe different to other micro:bit related
```
Comment (dpgeorge) [inline docs/i2c.rst]: maybe -> may be
While you're at it, can break the long line at "This".

**Comment by ntoll** [comment]
Ugh typos... rebased and fixed. Good spot x 2. ;-)

File: docs/i2c.rst [hunk 2]
```diff
@@ -8,6 +8,11 @@ using the I²C bus protocol. There can be multiple slave devices connected at
 the same time, and each one has its own unique address, that is either fixed
 for the device or configured on it. Your board acts as the I²C master.
 
+We use 7-bit addressing for devices because of the reasons stated
+`here <http://www.totalphase.com/support/articles/200349176-7-bit-8-bit-and-10-bit-I2C-Slave-Addressing>`_.
+
```
Comment (dpgeorge) [inline docs/i2c.rst]: Did you mean for this to be a separate paragraph?

Comment (ntoll) [inline docs/i2c.rst]: Yes. I think it important to accentuate that there's a possibly unexpected difference. The alternative was to emphasise it but it looked a bit naff IMHO.

**Comment by dpgeorge** [comment]
Rebase?

**Comment by ntoll** [comment]
Done.

**Comment by dpgeorge** [comment]
Awesome.

**Pull request merged**